src/axolotl/integrations/kd/chat_template.py

```diff
@@ -292,6 +301,14 @@ def _tokenize_single_prompt(self, prompt):
             tokenized_prompt["target_token_ids"] = target_token_ids
 
         return tokenized_prompt
```

Comment:
I think following the code flow now. We don't need this override in the V2 anymore. The hook handles this well!